src/pages/User/components/Modal/NotAMember.css

```diff
@@ -0,0 +1,48 @@
+.modal-wrapper {
+	background: white;
```

Comment:
Are we really using 8 spaces for css? Can you check other css files? It must be two.